administration/src/components/cards/CreateCardsController.tsx

```diff
@@ -34,6 +29,17 @@ const CreateCardsController = () => {
     )
   }
 
+  return <InnerCreateCardsController region={region} />
+}
+
+const InnerCreateCardsController = ({ region }: { region: Region }) => {
+  const projectConfig = useContext(ProjectConfigContext)
+  const client = useApolloClient()
+  const appToaster = useAppToaster()
+
+  const [cardBlueprints, setCardBlueprints] = useState<CardBlueprint[]>([projectConfig.createEmptyCard(region)])
+  const [state, setState] = useState(CardActivationState.input)
+
   const confirm = async () => {
     try {
       setState(CardActivationState.loading)
```

Comment:
Can you adjust the `ehrenamtskarten.pdf` file name below (to something like `berechtigungskarten.pdf`?)

Reply:
Will do!